lib/do_instance/do_metadata/do_set.js

```diff
@@ -119,7 +119,7 @@ do_set.synopses = ['{{name}} {{cmd}} [OPTIONS] INST [KEY=VALUE ...]',
                       '{{name}} {{cmd}} [OPTIONS] INST -f FILE'];
 
 do_set.help = [
-    'update one or more an instance.',
+    'Update one or more an instance.',
```

Comment:
Might as well go all the way on this and make it "Update one or more instances."